Tests/test_files/sodTest/sodTest.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
move this and the other notebook to a `Tools` directory, prefer using plain python scripts.  At the very least the less verbose jupyter output.